src/operators/mod.rs

```diff
@@ -70,6 +80,7 @@ pub(crate) fn reduce_epoch(
     _key: &TdPyAny,
     value: TdPyAny,
 ) {
+    debug!("{}, reducer:{:?}, key:{:?}, value:{:?}, agg:{:?}", log_func!(), reducer, _key, value, aggregator);
```

Comment:
Might be nice to also log out the updated aggregator after the reducer call, similar to stateful map.